docs/user-guide/modify-timing-data.ipynb

```diff
@@ -169,35 +172,40 @@
    "source": [
     "The `time_framework` details each benchmark year in which we run the simulation. The ```foresight``` variable details how much foresight an agent has when making investments. \n",
     "\n",
-    "As we have modified the timeslicing arrangements there will be a change in the underlying demand for heating. This may require more electricity to service this demand. Therefore, we relax the constraints for growth in the power sector for all technologies and constraints in the ```technodata/power/Technodata.csv```, as shown below:\n",
+    "We also have to modity the `forecast` value for each sector. For example, for the power sector the `sectors.power.subsectors.retro_and_new` section should now look as follows:\n",
+    "\n",
+    "    [sectors.power.subsectors.retro_and_new]\n",
+    "    agents = '{path}/technodata/Agents.csv'\n",
+    "    existing_capacity = '{path}/technodata/power/ExistingCapacity.csv'\n",
+    "    lpsolver = \"scipy\"\n",
+    "    forecast = 2\n",
```

Comment:
You could do this as a GitHub-style code block, i.e. starting with \`\`\`toml, so you get proper syntax highlighting.